.circleci/config.yml

```diff
@@ -1,13 +1,13 @@
 version: 2
 jobs:
-   build:
-     docker:
-       - image: circleci/python:3.6.4
-     steps:
-       - checkout
-       - run:
-           name: install tools
-           command: test/circle/install.sh
-       - run: 
-           name: lint
-           command: test/circle/lint.sh
+  build:
+    docker:
+     - image: unguiculus/chart-testing
```

Comment:
please pin a docker version

Reply:
could you also add the source of this docker image in the git repo?

Reply:
never mind, I found it in the PR

Reply:
The image will be updated to `gcr.io/kubernetes-charts-ci/test-image:v2.0.0` before the PR is merged.